log-elasticsearch/log-elasticsearch.cabal

```diff
@@ -1,5 +1,5 @@
 name:                log-elasticsearch
-version:             0.10.2.0
+version:             0.10.3.0
```

Comment:
Shouldn't this be a major version bump?

The behaviour of `elasticSearchLogger` has effectively changed, and following the [PVP decision tree](https://pvp.haskell.org/#decision-tree) suggests a major version bump.

Reply:
AFAIK the only thing that changes is that from now on the implicit mapping will have type of `insertion_order` be `long` instead of `integer`, which doesn't really matter.

But it might make sense to do it anyway.